frontend/src/components/Home.js

```diff
@@ -33,7 +37,8 @@ class Home extends React.Component {
       tractSelected: false,
       currentTract: {},
       geocoderValue: "",
-      displayGraph: false
+      displayGraph: false,
+      selectedDate: 25
```

Comment:
not sure if this is the best value for default selected date, if there's some way to automatically find the latest recorded response rate that would be better